build/jobs/cleanup.yml

```diff
@@ -0,0 +1,63 @@
+jobs:
+- job: DeleteResourceGroup
+  displayName: 'Delete resource group'
+  pool:
+    vmImage: 'vs2017-win2016'
+  steps:
+  - task: AzurePowerShell@4
+    displayName: 'Delete resource group'
+    inputs:
+      azureSubscription: 'Microsoft Health Open Source Subscription'
```

Comment:
Also variable

Reply:
Refactored

Reply:
Did this end up working as ${{ConnectedServiceName}}?

Reply:
Changed it everywhere except where you had originally commented...